### PR TITLE
Has been added the basic topics of: react-create-app

### PR DESCRIPTION
**Pull request by DmitriyRelkin**

### CODE REVIEW

**Comment by nesterone** [comment]
it's relatest to #1

File: content/react-create-app.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Getting started with Create React App
```
Comment (nesterone) [inline content/react-create-app.md]: Need to add metainfo for page, look on existing pages

File: content/react-create-app.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Theory
+1. [Create React App: Introduction](https://blog.logrocket.com/getting-started-with-create-react-app-d93147444a27/)
```
Comment (nesterone) [inline content/react-create-app.md]: add approximate time for reading each article

File: content/react-create-app.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+1. [Create React App: Quick Start](https://create-react-app.dev/docs/getting-started#creating-an-app)
+
+### Practice
+
+>Hint: If you already set up and has launched the project,
+>you may found already prepared main file in: /src/App.js
+
+Make any changes within jsx(html) structure, try for example to change
+the text within the paragraph tag, and make sure that these changes are displayed in the browser.
```
Comment (nesterone) [inline content/react-create-app.md]: Need info about repository location, few steps to do (fork, create pull requests)

**Comment by DmitriyRelkin** [comment]
@igornester PR is updated

**Comment by DmitriyRelkin** [comment]
@igornester PR is updated

**Pull request merged**